src/index.ts

```diff
@@ -434,15 +465,15 @@ class Spanner extends GrpcService {
         {
           name: formattedName,
           displayName: shortName,
+          nodeCount: config.nodes || 1,
```

Comment:
Is it possible the user would want `0`?

Reply:
[Docs](https://cloud.google.com/spanner/docs/instances#node_count)
`0` is not a valid value. The request with `node_count` as `0` will be rejected with 
`INVALID_ARGUMENT: Invalid CreateInstance request.` error.